### ISSUE TITLE
Limit amount of checkboxes

### ISSUE THREAD

**Issue by ColdstarJoey** [asmerkin/slideform#7]
Hey how do we limit the amount of checkboxes that can be checked, is it doable on the $form.slideform({
                    validate: {} section ?

**Comment by asmerkin** [comment]
@ColdstarJoey like you want to allow the user to check up to X of N checkboxes?

**Comment by ColdstarJoey** [comment]
yeah like if i have 5 checkboxes i only want the use to select 3 max
